apps/ui/src/helpers/argentx.ts

```diff
@@ -6,7 +6,19 @@ export default class Connector extends LockConnector {
     let provider;
     try {
       const argentx = await get();
-      const starknet = await argentx.connect();
+      const starknet = await argentx.connect({
+        dappName: 'Snapshot',
+        modalMode: localStorage.getItem('starknetLastConnectedWallet')
+          ? 'neverAsk'
+          : 'alwaysAsk',
+        modalTheme:
+          (localStorage.getItem('skin') as 'light' | 'dark') || 'system',
```

Comment:
Can we use `useUserSkin` there? We don't respect `system` preference in app theme, so it could be possible that this fallback won't match theme in app.
Also better if we have single source of truth.

Currently `system` fallback doesn't work either, because it's never falsy, it can be `none` though. If we use `useUserSkin` it will always be `light` or `dark` and always in-sync with app.

Reply:
Thank you! using useUserSkin now